xdsl/dialects/llvm.py

```diff
@@ -107,30 +107,112 @@ def is_typed(self):
 
 @irdl_op_definition
 class GEPOp(Operation):
+    """
+    The getelementptr is rather difficult to understand. There's a lot of
```

Comment:
Not sure I like this kind of docu. This is docu, not a blog :sweat_smile: 

Maybe remove this first paragraph and rewrite the rest slightly?

Reply:
In particular, I'd recommend removing the first person perspective, and making it technical and to the point. I'd also recommend adding a link to the MLIR/LLVM documentation in the header.